GETTING_STARTED_CLI.markdown

```diff
@@ -0,0 +1,294 @@
+This allows a client to test that a Veracruz instance is what it says it is.
+
+The Proxy Attestation Server uses an SQLite database. As a part of compilation,
+the proxy-attestation-server crate generates an empty database with the correct
+tables we can use for this examples.
+
+You can launch the Proxy Attestation Server with the
+`vc-pas`/`proxy-attestation-server` command. Note we are using the bash
+character `&` to launch the Proxy Attestation Server in the background:
+
+``` bash
+$ vc-pas example/example-policy.json --database-url=proxy-attestation-server/proxy-attestation-server.db &
+[2021-02-12T00:23:33Z INFO  proxy_attestation_server] Loaded policy 645ae94ea86eaf15cfc04c07a17bd9b6a3b3b6c3558fae6fb93d8ee4c3e71241
+[2021-02-12T00:23:33Z INFO  proxy_attestation_server] Using database "proxy-attestation-server/proxy-attestation-server.db"
+[2021-02-12T00:23:33Z INFO  actix_server::builder] Starting 12 workers
+[2021-02-12T00:23:33Z INFO  actix_server::builder] Starting "actix-web-service-127.0.0.1:3010" service on 127.0.0.1:3010
+[2021-02-12T00:23:33Z INFO  proxy_attestation_server] Proxy Attestation Server running on 127.0.0.1:3010
+```
+
+## Running the Veracruz Server
+
+The Veracruz Server is the main frontend for the compute side of Veracruz. It
+launches the Root Enclave (the trusted core of Veracruz) inside the enclave and
+provides a Rest API to communicating into the enclave.
+
+There is a lot of complexity underneath the hood, but launching the Veracruz
+Server requires only one command. Note again we are launching the Veracruz
+Server in the background:
+
+``` bash
+$ vc-server example/example-policy.json &
+[2021-02-12T00:55:40Z INFO  veracruz_server] Loading policy "example/example-policy.json"
+[2021-02-12T00:55:40Z INFO  veracruz_server] Loaded policy 645ae94ea86eaf15cfc04c07a17bd9b6a3b3b6c3558fae6fb93d8ee4c3e71241
+[2021-02-12T00:55:44Z INFO  actix_web::middleware::logger] 127.0.0.1:54762 "POST /Start HTTP/1.1" 200 96 "-" "-" 0.000542
+[2021-02-12T00:55:44Z INFO  actix_web::middleware::logger] 127.0.0.1:54764 "POST /SGX/Msg1 HTTP/1.1" 200 312 "-" "-" 0.233101
+[2021-02-12T00:55:45Z INFO  actix_web::middleware::logger] 127.0.0.1:54768 "POST /SGX/Msg3 HTTP/1.1" 200 25 "-" "-" 0.695685
+[2021-02-12T00:55:45Z INFO  actix_server::builder] Starting 12 workers
+[2021-02-12T00:55:45Z INFO  actix_server::builder] Starting "actix-web-service-127.0.0.1:3017" service on 127.0.0.1:3017
+[2021-02-12T00:55:45Z INFO  veracruz_server] Veracruz Server running on 127.0.0.1:3017
+```
+
+## Running the Veracruz Client 
+
+In practice you may integrate the `veracruz-client` Rust crate directly into
+your application, sending data programatically via the Rust API, but for this
+example we'll use the Veracruz Client CLI interface. This provides the same
+functionality, but in a CLI form.
+
+First lets send over the program to our Veracruz server, this requires an
+identity with the "PiProvider" role:
+
+<!-- TODO do we need - - target sgx? -->
```

Comment:
Probably the most reasonable. TZ is more complicated, Nitro is way harder to set up

Reply:
Ah, this was a more general question if vc-client needs --target at all (I couldn't type -- without messing up the markdown comments)

Reply:
Fixed after the CA merge, clients no longer need to know the target